README.md

```diff
@@ -2,14 +2,14 @@ The scripts in different folders are used to quickly build or manage the virtual
 
 Folders                | Description
 --------               | ---
-docker_builder         | Scripts to quickly build an compute node image on docker platform.
-kvm_builder            | Script to quickly build an compute node image on KVM platform.
+ansibleplaybook        | Playbook of ansible tasks for infrasim installation and basic commands.
+data_generater         | Script to generate the fru and sensor data for the new node simulation.
+deb_builder            | Scripts to build openipmi and qemu deb packages.
+diag_arp_flux          | Utility to fix ARP flux.
+dmidecode-2.12         | Utility to collect smbios data.
+docker                 | Contents to enable infrasim-compute to run in a docker container.
+fru_data_parser        | Lib to read and write emu file, aka, ipmi\_sim\_cmd set. Enables to operate FRU data command in human readable means.
 monorailtest           | Refer to the README.md in monorailtest folder.
-ova_builder            | Script to quickly build an OVA image.
+packer                 | Templates and related scripts that used for packer to build images.
 smbiostool             | Script to capture smbios data.
```

Comment:
Purpose of smbiostool is to change certain attributes in smbios file, not to capture smbios data. Please update @JuneZhou19